### ISSUE TITLE
Versions not showing on Top Nav bar on iOS

### ISSUE THREAD

**Issue by nate-double-u** [etcd-io/website#346]
## Issue
Versions menu not shown on top nav bar on iOS

## Additional info
![Image from iOS](https://user-images.githubusercontent.com/4453979/120362436-99a0d900-c2fa-11eb-9f19-1a9504e0afe6.png)
![Image from iOS (2)](https://user-images.githubusercontent.com/4453979/120362722-ea183680-c2fa-11eb-959f-5a39517cab00.png)

**Comment by chalin** [comment]
Just to be clear: we can't really tell from the screenshots because the top nav scrolls left and right on mobile (and there is nothing indicating whether there are more nav items on the left or the right).

I've checked this out myself on iOS and confirm that the versions menu isn't visible. This might be by design -- a docsy design decision. It would be worth checking that out.

**Comment by chalin** [comment]
Possibly related: https://github.com/google/docsy/issues/223#issuecomment-613750692

**Comment by nate-double-u** [comment]
> we can't really tell from the screenshots because the top nav scrolls left and right on mobile (and there is nothing indicating whether there are more nav items on the left or the right).

Interesting I didn't know about the scrolling (It may not be wide enough to scroll on my device, I'm not seeing that behavior). I'll remember to show other perspectives on mobile -- I'm also looking to confirm on other mobile OSs like Android.

Another iOS perspective:
![Image from iOS (3)](https://user-images.githubusercontent.com/4453979/120365527-0ff30a80-c2fe-11eb-9277-4de28171a86a.png)

**Comment by chalin** [comment]
> and there is nothing indicating whether there are more nav items on the left or the right

Of course, that's a docsy UX issue. Most folks don't know that the top nav scrolls. It took me a while before I noticed :).

Thanks for checking on Android.

**Comment by nate-double-u** [comment]
The problem also shows up on Android:

![signal-2021-06-01-110133_005](https://user-images.githubusercontent.com/4453979/120370504-2bf9aa80-c304-11eb-9ad1-d99213e9efb5.png)
![signal-2021-06-01-110133_004](https://user-images.githubusercontent.com/4453979/120370506-2c924100-c304-11eb-8d97-9308dcc6d31f.png)

**Comment by chalin** [comment]
Marking as an "upstream" problem (due to docsy) and giving it a p3-low priority since etcd's website readership from mobile is low and we might not get to this before the new IA is implemented.
